tjpcov/__init__.py

```diff
@@ -1,5 +1,13 @@
 # flake8: noqa
 from .covariance_builder import CovarianceBuilder
+from .covariance_fourier_gaussian_nmt import FourierGaussianNmt
+from .covariance_fourier_ssc import FourierSSCHaloModel
+from .covariance_cluster_counts import ClusterCounts
+from .covariance_gaussian_fsky import (
+    FourierGaussianFsky,
+    RealGaussianFsky,
+)
+from .wigner_transform import bin_cov, WignerTransform
```

Comment:
These imports were removed in #62. Leave `tjpcov/__init__.py` as in master.

Reply:
Will message on slack